aim/web/ui/src/components/Table/Table.scss

```diff
@@ -98,23 +98,30 @@
     align-items: center;
     width: 100%;
     justify-content: space-between;
+    position: relative;
     &.isDisabled {
       justify-content: flex-start;
     }
     &__deleteButton {
       display: none;
+      position: absolute;
+      top: 50%;
+      right: -1rem;
+      transform: translate(-50%, -50%);
```

Comment:
let's replace `right: -1rem` with `right: 0` and `transform: translate(-50%, -50%);` 
with `transform: translateY(-50%);`